source/documentation/administration_guide/topics/Uploading_the_VirtIO_Image_Files_to_a_Storage_Domain.adoc

```diff
@@ -1,11 +1,6 @@
 :_content-type: PROCEDURE
-[id="Uploading_the_VirtIO_and_Guest_Tool_Image_Files_to_an_ISO_Storage_Domain"]
-= Uploading the VirtIO and guest tool image files to a storage domain
-
-[NOTE]
-====
-The ISO domain is a deprecated storage domain type. The ISO Uploader tool has been removed in {virt-product-fullname} 4.4. {org-fullname} recommends uploading ISO images to the data domain with the Administration Portal or with the REST API. See xref:Uploading_Images_to_a_Data_Storage_Domain_storage_tasks[Uploading Images to a Data Storage Domain] for details.
-====
+[id="Uploading_the_VirtIO_Image_Files_to_aStorage_Domain"]
```

Comment:
I recommend that you either don't change the ID here, or be sure to search the documentation for references to it. Otherwise, cross references to this topic will be broken. I did a search and there is one xref to this topic in the file ../virtual_machine_management_guide/topics/Explanation_of_Settings_in_the_Run_Once_Window.adoc

Reply:
Thanks, fixed